proto/rill/admin/v1/api.proto

```diff
@@ -627,6 +659,55 @@ message GetUserResponse {
   User user = 1;
 }
 
+// Message containing user preferences
+message UserPreferences {
+  optional string timezone = 1;
```

Comment:
nit: should be `time_zone` (two words). This probably applies in other places as well (`TimeZone` instead of `Timezone)